userguide/dailyuse/waydroid.rst

```diff
@@ -0,0 +1,81 @@
+Android apps
+========================
+
+`Waydroid <https://waydro.id>`_ is a minimal Android container and compatibility layer to run Android apps on GNU/Linux operating systems such as Ubuntu Touch.
+
+.. note::
+    "Computer" refers to another device you connect your Ubuntu Touch device to (via USB here).
+    Your USB-attached computer must have ``adb`` and ``fastboot`` installed.
+
+.. note::
+    You will need to execute commands on your Ubuntu Touch device (and/or attached computer) to initialize Waydroid and install APKs.
+    You can do that by using the terminal application, but it is easier to use ``adb shell`` or :doc:`set up SSH </userguide/advanceduse/ssh>` to access your Ubuntu Touch device from your computer.
+
+
+How to set up
+-------------
+
+All Halium 9 and newer devices ship with waydroid out of the box starting Ubuntu Touch focal, if your device fits these requirements you can follow the folowing steps
+
+#. Run ``adb shell`` from your computer to get a shell on your Ubuntu Touch device (or open the terminal app on your ubuntu touch device).
+#. Run the following command on your Ubuntu Touch device: ``waydroid init``.
+
+Now you're done! You might have to refresh the app drawer (pull down from the center of the screen and release) for the new Android apps to show up.
+
+How to install new APKs
+-----------------------
+
+- Copy the APK to ``/home/phablet/Downloads``. Then run the following from your computer::
+
+    adb shell # Connect to your Ubuntu Touch device. You can also simply open the terminal app on your device.
+    waydroid app install /home/phablet/Downloads/my-app.apk
+    exit
+
+- Done! You might have to refresh the apps scope (pull down from the center of the screen and release) for the new Android apps to show up.
+
+Keep your apps up to date
+-------------------------
```

Comment:
This part confuses me a bit. We promote apk's for install, but once I have installed an app store like fdroid or amazon, I could also use that app store to *install* further apps, right. Which in fact is more convenient than the apk route ... How about changing "Keep your apps up to date" to "Use an app store in waydroid" and then explain that you could use one of the two and that would allow you the installation *and* update of apps

Reply:
Oh yeah true, will update

Reply:
uodated and elaborated